tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/connectors.py

```diff
@@ -121,6 +124,8 @@ def connectors(
         click.secho("No connector were selected according to your inputs. Please double check your filters.", fg="yellow")
         sys.exit(0)
 
+    ctx.obj["modified_connectors_in_commit"] = get_modified_connectors(ctx.obj["modified_files_in_commit"])
+    ctx.obj["modified_connectors_name_in_commit"] = [c.technical_name for c in ctx.obj["modified_connectors_in_commit"]]
```

Comment:
On publish, as we're on `master`, we care about the connectors that got modified in the `head` commits, not on the branch.